hubble/executor/hubio.py

```diff
@@ -978,6 +981,50 @@ def status(self) -> None:
             else:
                 console.log(f'Waiting `{task_id}` ...')
 
+    def _prettyprint_list_usage(self, console, executors, usage_kind=None):
+        from rich import box
+        from rich.panel import Panel
+        from rich.table import Table
+
+        param_str = Table(
+            box=box.SIMPLE,
+        )
+
+        param_str.add_column('executor')
+        param_str.add_column('path')
+
+        for index, item in enumerate(executors):
```

Comment:
```suggestion
        for item in executors:
```

Reply:
Yes, i will convert `for index, item in enumerate(executors):` to `for item in executors:`